Git Formats/Git Log.sublime-syntax

```diff
@@ -13,13 +13,33 @@ contexts:
   main:
     # 1st header line
     # commit d9d9fb804f5d61c13ba2f8746af33a9f3c609075
-    - match: (?:(commit)\s+(\h{7,}))?\s*\n
+    - match: \b(commit)\s+(\h{7,})
```

Comment:
`\b` is not really required, is it?

`commit` is the first word in a log and `commit-header` pops only if it sees `commit`.

Reply:
it's maybe not required, but does it do any harm? for context, I also plan to make another PR to fix `git show` diffs which are currently unhighlighted, and felt it would be clearer with the `\b`. (See https://github.com/sharkdp/bat/issues/1735 for context.)